episodes/5-transfer-learning.Rmd

```diff
@@ -17,14 +17,16 @@ Instead of training a model from scratch, with transfer learning you make use of
 
 An example: Let's say that you want to train a model to classify images of different dog breeds. You could make use of a pre-trained network that learned how to classify images of dogs and cats. The pre-trained network will not know anything about different dog breeds, but it will have captured some general knowledge of, on a high-level, what dogs look like, and on a low-level all the different features (eyes, ears, paws, fur) that make up an image of a dog. Further training this model on your dog breed dataset is a much easier task than training from scratch, because the model can use the general knowledge captured in the pre-trained network.
 
+![](episodes/fig/05-transfer_learning.png)
+<!-- # TODO: perhaps the Mermaid markdown can be embedded directly in the Rmd file? -->
```

Comment:
I like how we solve this in https://github.com/carpentries-incubator/python-intermediate-development/blob/da6fc57abee81115403dacce5ffc544ecf595a36/episodes/10-section1-intro.md?plain=1#L36-L54

We add a comment with instructions and everything you need to adapt the figure with mermaid.

Perhaps we can also do it in this way?

Reply:
Really nice to add this picture by the way!